src/pages/Chains/RegionalNav.tsx

```diff
@@ -117,99 +109,16 @@ export type ChainResource = Resource<Chain>
 interface Props extends WithStyles<typeof styles> {
   chainId: string
   chain?: ChainResource
-  deleteChain: (...args: any[]) => any
 }
 
-const DeleteSuccessNotification = ({ id }: any) => (
-  <React.Fragment>Successfully deleted chain {id}</React.Fragment>
-)
-
-const RegionalNavComponent = ({
-  classes,
-  chainId,
-  chain,
-  deleteChain,
-}: Props) => {
-  const [modalOpen, setModalOpen] = useState(false)
-  const [deleted, setDeleted] = useState(false)
+const RegionalNavComponent = ({ classes, chainId, chain }: Props) => {
   const location = useLocation()
   const navOverridesActive = location.pathname.endsWith('/config-overrides')
   const editActive = location.pathname.endsWith('/edit')
   const navNodesActive = !navOverridesActive && !editActive
 
-  const handleDelete = (id: string) => {
-    deleteChain(id, () => DeleteSuccessNotification({ id }), ErrorMessage)
-    setDeleted(true)
-  }
-
   return (
     <>
-      <Dialog
```

Comment:
So we want to delete this dialog? No more delete chain is it?
Is it not used?

Reply:
Chains can now only be created or updated by changing the chain TOML. Long time ago all that config was stored in the DB and was dynamic. A previous PR removed most of the UI (it's already not visible) but a bunch of leftovers remained

Reply:
ah good context here, i didn know that, thanks!